analyzer-comments/elixir/solution/no_rescue.md

```diff
@@ -0,0 +1,25 @@
+# no rescue
+
+Do not use exceptions for flow control. Try to solve this exercise without using `rescue`.
+
+If you're using a standard library function whose name ends with `!` (raises errors), find an equivalent function.
+
+Note: If you're seeing this automated comment on your solution, the other automated comments might not be accurate. Fix this problem first!
+
+```elixir
+# preferred: for example `case`
+
+case Date.new(1999, 13, 1) do
+  {:ok, date} -> date
+  {:error, _} -> nil
+end
+
+# discouraged: rescuing exceptions
```

Comment:
Should we also mention that something like this would raise
```elixir
{:ok, date} = Date.new(1999, 13, 1)
```
Or does it feel like a different topic?

Reply:
It does feel like a different topic. This comment is only triggered by usages of `rescue`, it doesn't really matter how the error is raised... the goal is to prevent `rescuing`.